lua/null-ls/info.lua

```
@@ -30,7 +31,7 @@ M.show_window = function()
 
     local bufnr = api.nvim_get_current_buf()
     if not client or not lsp.buf_is_attached(bufnr, client.id) then
-        u.echo("WarningMsg", "failed to get info: buffer is not attached")
+        Log:warn("failed to get info: buffer is not attached")
```

I think this should always output to `messages` regardless of the user's log settings, since it's easy to imagine a situation where the command fails silently.

---

A user would need to explicitly turn off the console option, and we can even avoid exposing that option altogether.

By the way, the same logger logic is used by Packer, Telescope and quite a few others.